_overviews/hello-scala/case-objects.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: multipage-overview
+title: Case Objects
+description: This lesson introduces Scala 'case objects', which are used to create singletons with a few additional features.
+partof: hello_scala
+overview-name: Hello, Scala
+discourse: true
+num: 52
+previous-page: case-classes
+next-page: functional-error-handling
+---
+
+
+Before we jump into *case objects*, we should provide a little background on “regular” Scala objects. As we mentioned early in this book, you use a Scala `object` when you want to create a singleton object. As [the official Scala documentation states](https://docs.scala-lang.org/tour/singleton-objects.html), “Methods and values that aren’t associated with individual instances of a class belong in singleton objects, denoted by using the keyword `object` instead of `class`.”
```

Comment:
```suggestion
Before we jump into *case objects*, we should provide a little background on “regular” Scala objects. As we mentioned early in this book, you use a Scala `object` when you want to create a singleton object. As [documentation states]({{site.baseurl}}/tour/singleton-objects.html), “Methods and values that aren’t associated with individual instances of a class belong in singleton objects, denoted by using the keyword `object` instead of `class`.”
```

Reply:
I resolved this in the code (text).